cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerDeployClientConfigProvider.java

```diff
@@ -135,30 +146,30 @@ BigDecimal getCommandIdFromActiveCommands(ApiResponse<ApiCommandList> response)
     }
 
     @VisibleForTesting
-    BigDecimal getCommandIdFromCommandsTable(ClustersResourceApi api,
+    BigDecimal getCommandIdFromCommandsTable(ClustersResourceApi api, String path,
             Map<String, List<String>> headers) throws CloudbreakException, ApiException {
         try {
             OkHttpClient httpClient = api.getApiClient().getHttpClient();
-            Request request = createRequest(api.getApiClient(), headers);
+            Request request = createRequest(api.getApiClient(), path, headers);
             Response response = httpClient
                     .newCall(request)
                     .execute();
             if (response.code() >= ERROR_CODES_FROM) {
                 LOGGER.debug("{} request against Cloudera Manager API returned with status code: {}, response: {}",
-                        CMF_COMMANDS_COMMAND_TABLE_URI_PATH, response.toString(), response.code());
+                        path, response.toString(), response.code());
                 throw new CloudbreakException(
                         String.format("%s request against CM API returned with status code: %d",
-                                CMF_COMMANDS_COMMAND_TABLE_URI_PATH, response.code()));
+                                path, response.code()));
             }
-            return getCommandIdFromCMResponse(response);
+            return getCommandIdFromCMResponse(response, path);
         } catch (IOException e) {
```

Comment:
if there is any exception is received which is not `IOException` during `CMF_COMMANDS_ACTIVE_COMMAND_TABLE_URI_PATH` call, we won't try it with `CMF_COMMANDS_RECENT_COMMAND_TABLE_URI_PATH`. Is it expected to behave like this?

Reply:
yes, if the call is broken (bad response etc.) I would expect the other one as well... also if let's say activeCommandTable fails, that does not mean that deployClusterClientConfig command will be found in commandTable